tools/generate.js

```diff
@@ -158,6 +164,7 @@ let mappings = {
 
   "color: #0366d6": "color: /*[[base-color]]*/ #4f8cc9", // needs to be after #333
   "color: #1074e7": "color: /*[[base-color]]*/ #4f8cc9",
+  "color: #1F61A0": "color: /*[[base-color]]*/ #4f8cc9", //  graphql explorer
```

Comment:
@silverwind are these values case sensitive? A few of the colors in this style sheet use all caps which is why they're using it here too.

Reply:
I dont change the hex case from the default stylesheets when adding them to generator, though I always use lower case for replacement colors as its easier to read 8 vs B etc.

Reply:
They are compared insensitively, thought I do prefer lowercase in this file.